### PR TITLE
Add simdjson to build summary

### PR DESCRIPTION
**Pull request by dominiklohmann**
###  :notebook_with_decorative_cover: Description

This adds simdjson to the build summary. Also, this greatly improves the dependency_summary CMake function.

###  :memo: Checklist

- [x] All user-facing changes have changelog entries.
- [x] The changes are reflected on [docs.tenzir.com/vast](https://docs.tenzir.com/vast), if necessary.
- [x] The PR description contains instructions for the reviewer, if necessary.

### :dart: Review Instructions

Review the CMake code, check CI or run locally to see output.

### CODE REVIEW

File: cmake/VASTUtilities.cmake [hunk 1]
```diff
@@ -57,10 +57,17 @@ endmacro (install_symlink)
 macro (dependency_summary name what)
   get_property(VAST_DEPENDENCY_SUMMARY GLOBAL
                PROPERTY VAST_DEPENDENCY_SUMMARY_PROPERTY)
-  if ("${what}" STREQUAL "" OR "${what}" MATCHES "-NOTFOUND$")
+  if (TARGET ${what})
+    get_target_property(location "${what}" LOCATION)
+    get_filename_component(location "${location}" PATH)
+    get_filename_component(location "${location}" PATH)
```
Comment (tobim) [inline cmake/VASTUtilities.cmake:63]: According to TFM:
```
PATH      = Legacy alias for DIRECTORY (use for CMake <= 2.8.11)
```
```suggestion
    get_filename_component(location "${location}" DIRECTORY)
    get_filename_component(location "${location}" DIRECTORY)
```

**Review by tobim — APPROVED** [review]
Great improvement! Please use the target version for `jemalloc` as well.

File: cmake/VASTUtilities.cmake (hunk 1 above)
Comment (dominiklohmann) [reply cmake/VASTUtilities.cmake]: Nice catch.

**Comment by dominiklohmann** [comment]
[ch4981]

**Pull request merged**